lib/ruby/sablon.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class Curated::Sablon < Curated::RubyGem
+  package 'sablon'
+  homepage 'https://github.com/senny/sablon'
+  category Category::Other
+
+  text =
+    '# First prepare docx file and add `MergeField` inside. For example the name of this MergeField is some_variable' \
+    "\n\nrequire 'sablon'\n\n template = Sablon.template(File.expand_path('/Users/bs/Documents/example_merge_field.docx'))" \
+    "\ncontext = { some_variable: 'This text will be added instead of `some_variable` in resulted docx file' }" \
+    "\n\ntemplate.render_to_file File.expand_path('~/Users/bs/Documents/output.docx'), context"
+
+  pros "We've been usi Sablon in some projects for inserting content in docx templates."
+  pros "It is easy to use this gem:\n #{Utils.ruby_example_code(text)}"
```

Comment:
Please avoid using Utils directly in Markdown. Better create anonymous class that encapsulates all necessary helpers in `Curated` class. You can take a look at similar definition here: https://github.com/jetrockets/curated/blob/main/lib/curated/renderer.rb#L15.

Reply:
Thanks for the new thoughts. May you please see the new implementation?